subsys/bluetooth/controller/ll_sw/ull_sync.c

```diff
@@ -1080,17 +1080,17 @@ void ull_sync_chm_update(uint8_t sync_handle, uint8_t *acad, uint8_t acad_len)
 int ull_sync_slot_update(struct ll_sync_set *sync, uint32_t slot_plus_us,
 			 uint32_t slot_minus_us)
 {
+	uint32_t volatile ret_cb;
```

Comment:
I am unsure what this change has to do with time reservations. Can you elaborate?

Reply:
But the `volatile` in context of `ticker_updat_op_status_give` is dropped in line https://github.com/zephyrproject-rtos/zephyr/blob/5346d3e1495a6d2406fd76927a97a32e4eeeaf2f/subsys/bluetooth/controller/ll_sw/ull_sync.c#L1096. Compiler will take into account the `volatile` keyword only in context of this function.

Reply:
Refer to: https://github.com/zephyrproject-rtos/zephyr/blob/5346d3e1495a6d2406fd76927a97a32e4eeeaf2f/subsys/bluetooth/controller/ll_sw/ull_sync.c#L1328-L1334

The `param` is applied the `volatile` qualifier when updated.

The change in this line is to convey that `ret_cb` is updated outside the current execution context (in ISR in this case).